codex-rs/core/src/rollout.rs

```diff
@@ -321,3 +339,43 @@ async fn rollout_writer(
         }
     }
 }
+
+pub async fn prepare_rollout_recorder(
+    config: &Config,
+    session_id: Uuid,
+    instructions: Option<String>,
+    resume_path: Option<&Path>,
+) -> RolloutSetup {
+    // Try to resume
+    let (mut restored_items, mut recorder_opt) = (None, None);
```

Comment:
Can you please make these two separate declarations? I don't feel like this is idiomatic Rust.